sys/include/net/gnrc/ipv6/nib/abr.h

```diff
@@ -45,7 +58,59 @@ int gnrc_ipv6_nib_abr_add(const ipv6_addr_t *addr);
  * @param[in] addr  The address of an authoritative border router.
  */
 void gnrc_ipv6_nib_abr_del(const ipv6_addr_t *addr);
```

Comment:
Let's make this `int` so it may also 'return' `-ENOTSUP` if it's not implemented. (and another error code if the address can not be found)

Reply:
> (and another error code if the address can not be found)

None of the other data structures do this and it also just adds unnecessary overhead. Why is it important to know if it is not found for removal?

Reply:
> Let's make this `int` so it may also 'return' `-ENOTSUP` if it's not implemented.

Mhhh... Not convinced. This is purely user-specific cosmetics. I can however edit adopt the shell command to return an error if the command is not supported ;-)

Reply:
Done

Reply:
It's not, I just thought if the function can report an error anyway, might as well make use of it.

But if this is not idiomatic, I'm alwaysfor less overhead :)

I already wanted to suggest to just not advertise any function other than `show` for the !6BR case when they are doing nothing. Those debug strings tend to slowly eat up ROM over time.

Reply:
> I already wanted to suggest to just not advertise any function other than `show` for the !6BR case when they are doing nothing. Those debug strings tend to slowly eat up ROM over time.

Good idea, done (however for shell commands its less of an issue to be waste full ;-P).